graph2tac/loader/predict_server.py

```diff
@@ -609,10 +618,9 @@ def prediction_loop(self, context: GlobalContextMessage):
 
                     self.response_history.record_response(response)
 
-                    # sending response back to coq
-                    context.prediction_requests.send(response)
-
-                    self.log_cnts.measure_t_coq_start()
+                    with self.log_cnts.measure_t_coq_send():
+                        # sending response back to coq
+                        context.prediction_requests.send(response)
```

Comment:
While I like the context manager approach, I don't think it is good to basically put the old `measure_t_coq_start()` inside the context manager for something else.  I think leaving `measure_t_coq_start()` as it's own thing is more clear.

Reply:
ok, I will make it separate

Reply:
One of the reasons I put it inside was actually that I wanted just a single counter `n_coq`. There is also an alternative to get rid of that counter entirely and use just the step index.